src/main/java/quickcarpet/settings/Settings.java

```diff
@@ -104,6 +104,9 @@
     @Rule(category = {FEATURE, EXPERIMENTAL})
     public static boolean movableBlockEntities = false;
 
+    @Rule(category = {FEATURE,EXPERIMENTAL})
+    public static boolean movableBlockOverwrites = false;
```

Comment:
Override would be the better word, I think: https://english.stackexchange.com/a/88403

Reply:
Probably you are right

Reply:
Can you do that? If not I can do it after merging.

Reply:
done